ui-v2/app/templates/dc/services/instance.hbs

```diff
@@ -24,44 +24,19 @@
         <dt>Node Name</dt>
         <dd><a href="{{href-to 'dc.nodes.show' item.Node.Node}}">{{item.Node.Node}}</a></dd>
       </dl>
-{{#if proxy.ServiceName}}
-      <dl>
-      </dl>
-{{/if}}
-{{#if (eq item.Kind 'connect-proxy')}}
-  {{#if item.Proxy.DestinationServiceID}}
-      <dl>
-        <dt data-test-proxy-destination="instance">Dest. Service Instance</dt>
-        <dd><a href="{{href-to 'dc.services.instance' item.Proxy.DestinationServiceName item.Node.Node item.Proxy.DestinationServiceID}}">{{item.Proxy.DestinationServiceID}}</a></dd>
-      </dl>
-      <dl>
-        <dt>Local Service Address</dt>
-        <dd>{{item.Proxy.LocalServiceAddress}}:{{item.Proxy.LocalServicePort}}</dd>
-      </dl>
-  {{else}}
-      <dl>
-        <dt data-test-proxy-destination="service">Dest. Service</dt>
-        <dd><a href="{{href-to 'dc.services.show' item.Proxy.DestinationServiceName}}">{{item.Proxy.DestinationServiceName}}</a></dd>
-      </dl>
-  {{/if}}
-{{/if}}
     </BlockSlot>
     <BlockSlot @name="content">
         <TabNav @items={{
           compact
               (array
                       (hash label="Health Checks" href=(href-to "dc.services.instance.healthchecks") selected=(is-href "dc.services.instance.healthchecks"))
-(if
-  (eq item.Kind 'connect-proxy')
-                      (hash label="Upstreams" href=(href-to "dc.services.instance.upstreams") selected=(is-href "dc.services.instance.upstreams")) ""
-)
-(if
-  (and (eq item.Kind 'connect-proxy') (gt item.Proxy.Expose.Paths.length 0))
-                      (hash label="Exposed Paths" href=(href-to "dc.services.instance.exposedpaths") selected=(is-href "dc.services.instance.exposedpaths")) ""
-)
 (if
   (eq item.Kind 'mesh-gateway')
                       (hash label="Addresses" href=(href-to "dc.services.instance.addresses") selected=(is-href "dc.services.instance.addresses")) ""
+)
```

Comment:
This reminds me I think the Addresses tab probably also needs updating to the new composite-row design? @opihana I just took a quick look at the designs for this round and this tab isn't in there at all? Do we need to add that in there with the new skin?

Reply:
There is a new Address Tab frame. I will work on this in another PR.